plugins/woocommerce-beta-tester/src/live-branches/components/BranchList.tsx

```diff
@@ -2,50 +2,59 @@
  * External dependencies
  */
 import {
-	// @ts-ignore
-	__experimentalItemGroup as ItemGroup,
-	// @ts-ignore
-	__experimentalItem as Item,
 	Button,
+	ComboboxControl,
 } from '@wordpress/components';
```

Comment:
Do you have prettier set up in your environment or are you manually formatting? When prettier formats my files it makes this a one-liner amongst a few other changes.

Let me know if you need some help ensuring that prettier runs on file save.